test/src/api/commands/web-element/testIsPresent.js

```diff
@@ -0,0 +1,88 @@
+const assert = require('assert');
+const {WebElement} = require('selenium-webdriver');
+const MockServer = require('../../../../lib/mockserver.js');
+const CommandGlobals = require('../../../../lib/globals/commands-w3c.js');
+const common = require('../../../../common.js');
+const Element = common.require('element/index.js');
+
+describe('element().isPresent() command', function() {
+  before(function(done) {
+    CommandGlobals.beforeEach.call(this, done);
+  });
+
+  after(function(done) {
+    CommandGlobals.afterEach.call(this, done);
+  });
+
+  it('test .element().isPresent() for element present', async function() {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element',
+      method: 'POST',
+      response: JSON.stringify({
+        value: {ELEMENT: '0'} // Simulate element found
+      })
+    }, true);
```

Comment:
This mock is not useful because that URL is never hit.

To see all the request/response happening behind the scenes, run the test as `VERBOSE=1 npx mocha test/src/api/commands/web-element/testIsPresent.js`.

Reply:
How to get more information about the mockserver? I will try to refer [this](https://w3c.github.io/webdriver/#endpoints) as you have suggested in other PR.

Reply:
MockServer is mostly fairly simple, read this: https://discord.com/channels/618399631038218240/1070690424185966662/1210478133371871243.

It just intercepts the calls going on a particular URL and returns the response provided by us.

Reply:
Hello @garg3133 , this is the verbose output I am getting after removing the mock servers and I am finding the URL to be the same.
`
  element().isPresent() command
⠋ Connecting to localhost on port 10195...
   Request POST /session  
   {
     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: 'firefox' } }
  }
   Response 201 POST /session (13ms)
   {
     value: {
       sessionId: '13521-10219-202',
       capabilities: {
         acceptInsecureCerts: false,
         browserName: 'firefox',
         browserVersion: '65.0.1',
         platformName: 'linux',
         platformVersion: '4.9.125-linuxkit',
         setWindowRect: true,
         strictFileInteractability: false,
         timeouts: { implicit: 0, pageLoad: 300000, script: 30000 },
         unhandledPromptBehavior: 'dismiss and notify'
       }
     }
ℹ Connected to localhost on port 10195 (19ms).
  Using: firefox (65.0.1) on LINUX (4.9.125-linuxkit).

 Received session with ID: 13521-10219-202

 
 → Running command: element.find ()
   Request POST /session/13521-10219-202/elements  
   { using: 'css selector', value: '#signupSection' }
   Response 200 POST /session/13521-10219-202/elements (1ms)
   { value: [ { 'element-6066-11e4-a52e-4f735466cecf': '0' } ] }
 → Completed command: element.find () (2ms)
 
 → Running command: element().isElementPresent ()
    ✔ test .element().isPresent() for element present
 → Completed command: element().isElementPresent () (0ms)
 
 → Running command: element.find ()
   Request POST /session/13521-10219-202/elements  
   { using: 'css selector', value: '#signupSecti' }
   Response 404 POST /session/13521-10219-202/elements (1ms)

  UnsupportedOperationError
   findElements: 
 → Completed command: element.find () (2ms)
 
 → Running command: element().isElementPresent ()
    ✔ test .element().isPresent() for element not present
 → Completed command: element().isElementPresent () (0ms)
 
 → Running command: element.find ()
   Request POST /session/13521-10219-202/elements  
   { using: 'css selector', value: '#signupSection' }
   Response 200 POST /session/13521-10219-202/elements (0ms)
   { value: [ { 'element-6066-11e4-a52e-4f735466cecf': '0' } ] }
 → Completed command: element.find () (2ms)
 
 → Running command: element().isElementPresent ()
    ✔ test async .element().isPresent() for element present
 → Completed command: element().isElementPresent () (0ms)
 
 → Running command: element.find ()
   Request POST /session/13521-10219-202/elements  
   { using: 'css selector', value: '#badElement' }
   Response 404 POST /session/13521-10219-202/elements (0ms)

  UnsupportedOperationError
   findElements: 
 → Completed command: element.find () (2ms)
 
 → Running command: element().isElementPresent ()
    ✔ test .element().isPresent() handles errors
 → Completed command: element().isElementPresent () (0ms)`

Reply:
Hello @garg3133 , `isPresent` is working fine after removing the mock servers so do we actually need mock servers for the `isPresent` command or should I remove this completely? I have taken reference from [here](https://github.com/nightwatchjs/nightwatch/blob/main/test/src/api/commands/element/testIsPresent.js)and they are also not using mock servers. Also, can you please review my PR?